extensions/NexusKitten/sgrab.js

```diff
@@ -0,0 +1,249 @@
+
+    class SGrab {
+      getInfo() {
+        return {
+          id: 'sgrab',

```

Comment:
change the extension ID to somehow incorporate your username eg. `nexuskittensgrab`

Reply:
Alright, should be changed now